src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,56 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ITEMS_NUMBER = 10;
+    private static final int FIRST_INDEX_OF_ARRAY = 0;
+    private int indexOfEmptyElement;
```

Comment:
```suggestion
    private static final int MAX_ITEMS_NUMBER = 10;
    private int indexOfEmptyElement;
```
you don't need to set it, it's int so it will be set to 0 by default